package.json

```diff
@@ -48,6 +48,21 @@
           "type": "boolean",
           "default": false,
           "description": "Cards will pile up until they are acknowledge by the user."
+        },
+        "zvRandomCards.useWeight": {
+          "type": "boolean",
+          "default": false,
+          "description": "The pile will be shown aggregated and the acknowledgment will acknowledge the whole pile"
+        },
+        "zvRandomCards.aggregatePile": {
```

Comment:
Update line 121:
```
{
  "command": "zvRandomCards.resetPile",
  "when": "view == zvRandomCards.cardView && config.zvRandomCards.pileUp && !config.zvRandomCards.aggregatePile",
  "group": "navigation"
}
```

so we don't see the reset pile button when we enable aggregate (since acknowledge clears the whole pile)

Reply:
Foooort